waspc/cli/Command/Start.hs

```diff
@@ -24,12 +24,6 @@ start = do
   waspRoot <- findWaspProjectRootDirFromCwd
   let outDir = waspRoot </> Common.dotWaspDirInWaspProjectDir </> Common.generatedCodeDirInDotWaspDir
 
-  waspSaysC "Compiling wasp code..."
```

Comment:
Here I've moved this bit after the setup, so I could add another `do` block and pass the `wasp` result to `watch`. Not sure if this was the best way to do it though 😅

Reply:
Super nice breakdown! Just updated the order of execution back to normal, so clear now. I think this was the reason why I was also not getting the expected result when trying to run the setup again.

Reply:
Super nice breakdown! Just updated the order of execution back to the correct order. I remember I tried to run `setup` and it wasn't working as expected, and the order of execution was probably the reason now I think about it. Thanks!

Reply:
Great :)!